addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -691,6 +704,9 @@ public String match(String contents) {
 
     private static class DirNamesContentsMatcher implements ContentsMatcher {
 
+        public static final List<String> NIX_MATCHES =
```

Comment:
Could well convert to string here instead of each time when raising the alerts, also could drop the [].